CHANGELOG.md

```diff
@@ -11,6 +11,107 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.6.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.6.0. This release brings new features such as:
+- Several building blocks for GPU-resident sparse direct solvers like symbolic
+  and numerical LU and Cholesky factorization, symbolic Cholesky factorization,
+  numerical LU factorization, ...,
+- A distributed Schwarz preconditioner,
+- New FGMRES and GCR solvers,
+- Distributed benchmarks for the SpMV operation, solvers, ...
+- Support for non-default streams in the CUDA and HIP backends,
+- Mixed precision support for the CSR SpMV,
+- A new profiler logger which integrates with NVTX, ROCTX, TAU and VTune to
+  provide internal Ginkgo knowledge to most HPC profilers!
+
+and much more.
+
+If you face an issue, please first check our [known issues page](https://github.com/ginkgo-project/ginkgo/wiki/Known-Issues) and the [open issues list](https://github.com/ginkgo-project/ginkgo/issues) and if you do not find a solution, feel free to [open a new issue](https://github.com/ginkgo-project/ginkgo/issues/new/choose) or ask a question using the [github discussions](https://github.com/ginkgo-project/ginkgo/discussions).
+
+Supported systems and requirements:
++ For all platforms, CMake 3.13+
++ C++14 compliant compiler
++ Linux and macOS
+  + GCC: 5.5+
+  + clang: 3.9+
+  + Intel compiler: 2018+
+  + Apple LLVM: 8.0+
```

Comment:
No idea. Only mac users can answer this. We probably should check it manually or bump it.

Reply:
Yeah, I think we should just bump it. It is a pain to verify it as I think Apple forces you to install XCode for even specific Apple Clang versions, which is upwards of 10GB. In case someone is able to compile and run it on their machine for lower versions, then that is fine.